src/frontend/src/lib/api/backend.api.ts

```diff
@@ -102,3 +108,28 @@ export const setUserToken = async ({
 	const { set_user_token } = await getBackendActor({ identity });
 	return set_user_token(token);
 };
+
+export const createUserProfile = async ({
+	identity
+}: {
+	identity: Identity;
+}): Promise<UserProfile> => {
+	const { create_user_profile } = await getBackendActor({ identity });
+	return create_user_profile();
+};
+
+export const getUserProfile = async ({
+	identity,
+	certified = true
+}: { identity: Identity } & QueryParams): Promise<UserProfile> => {
+	const { get_user_profile } = await getBackendActor({ identity, certified });
+	const response = await get_user_profile();
```

Comment:
I would maybe implement interpreting the response in a service and keep the function "dumb"? The api is basically just a wrapper proxy.

Reply:
Plus it allows to internationalize error message and avoid hardcoding "Unknown error" text.

Reply:
Done! I wasn't sure either which pattern you preferred.